__tests__/app.test.js

```diff
@@ -0,0 +1,56 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const app = require("../app.js");
+const request = require("supertest");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("GET/api/topics", () => {
+  test("200 and responds with an array of topic objects each of which should have the following properties slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
+  test("200: returns the correct object key - value pairs", () => {
```

Comment:
You could remove this test as the previous test ensures that ALL items coming back are the correct shape.